frontend/components/FleetAce/FleetAce.tsx

```diff
@@ -47,22 +45,12 @@ const FleetAce = ({
   onLoad,
   onChange,
   handleSubmit = noop,
-}: IFleetAceProps) => {
+}: IFleetAceProps): JSX.Element => {
   const editorRef = useRef<ReactAce>(null);
   const wrapperClass = classnames(wrapperClassName, baseClass, {
     [`${baseClass}__wrapper--error`]: !!error,
   });
 
-  const [isReady, setIsReady] = useState<boolean>(false);
-
-  useEffect(() => {
-    setTimeout(() => {
-      setIsReady(true);
-    }, 0);
```

Comment:
I could not find a reason for this 0ms timeout. Presumably it was used to hold the page render until the global config had been applied. No longer needed with the config fix.